src/SearchResults.js

```diff
@@ -0,0 +1,185 @@
+import React, { useState } from "react";
+import CustomerProfile from "./CustomerProfile";
+import moment from "moment";
+
+const useSortableData = (items, config = null) => {
+  const [sortConfig, setSortConfig] = useState(config);
+
+  const sortedItems = React.useMemo(() => {
+    let sortableItems = [...items];
+    if (sortConfig !== null) {
+      sortableItems.sort((a, b) => {
+        if (a[sortConfig.key] < b[sortConfig.key]) {
+          return sortConfig.direction === "ascending" ? -1 : 1;
+        }
+        if (a[sortConfig.key] > b[sortConfig.key]) {
+          return sortConfig.direction === "ascending" ? 1 : -1;
+        }
+        return 0;
+      });
+    }
+    return sortableItems;
+  }, [items, sortConfig]);
+
+  const requestSort = key => {
+    let direction = "ascending";
+    if (
+      sortConfig &&
+      sortConfig.key === key &&
+      sortConfig.direction === "ascending"
+    ) {
+      direction = "descending";
+    }
+    setSortConfig({ key, direction });
+  };
+
+  return { items: sortedItems, requestSort, sortConfig };
+};
+
+const SearchResults = props => {
+  const [isActive, setIsActive] = useState(null);
+  const [index, setIndex] = useState("");
+  const { bookings } = props;
+  let sortedResults = [...bookings];
+  console.log(sortedResults);
+
+  const { items, requestSort } = useSortableData(props.bookings);
+
+  const toggleActive = i => {
+    //Remove the if statement if you don't want to unselect an already selected item
+    if (i === isActive) {
+      setIsActive(isActive => (isActive = null));
```

Comment:
I'm not sure this is quite correct - when we pass a function `setIsActive()`, then the _return_ value is what is set in state. What should we do to fix this?

Reply:
Yes that makes sense it would be same  even if I just write setIsActive(null)